### DISCUSSION TITLE
i Caught a Bug in Budibase template

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10708] [Help] [answered]
on Budibase online website i imported professional directory template without any changes. i published it.

Now when i go to see its public pages then it gives session not authenticated error. I didn't touch anything in this template. 

Permissions for screen  for public is giving error even on Budibase templates on Budibase.app website.

any solution?

Here is the screenshot 

![16850856043441864851828308899139](https://github.com/Budibase/budibase/assets/130876667/c6a34e8d-3b42-4f16-b2c5-7b5430ba59b8)

**Comment by sghriyadh** [comment]
It seems problem is in public level access.

i changed another template to full public but still th e same "session not authenticated" error 

![16851729422173236676841561079195](https://github.com/Budibase/budibase/assets/130876667/55e0fe56-f061-4f61-b385-39e9bc23677b)


![Uploading 16851729876922306039413089473937.jpg…]()

**Comment by melohagan** [comment, marked as answer]
Hey @sghriyadh 

This is because the access level of the Directory table is **Basic**. It should be **Public** in order to work for public users. 

This a mistake in the app template, and I have raised a bug here: https://github.com/Budibase/budibase/issues/10773
